### DISCUSSION TITLE
What does the Git Clone metric mean in the traffic section of a repo?

### DISCUSSION THREAD

**Discussion by dhyani15** [community/community#23994] [New to GitHub]
<p>Hey folks,</p>
<p>I am new to GitHub and was hoping to understand what does the Git Clone metric mean in the traffic section of a repo? Like what does it indicate?</p>
<p>Thanks for any input!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>The git clone metric is simply a count of how many times the repository has been <a href="https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository">cloned</a>.</p>

**Reply by AbdelrahimAlsadiq** [reply]
did you figure it out yourself?@?@!?!?!?!
